Tests/Doctrine/DBAL/Types/PhoneNumberTypeTest.php

```diff
@@ -41,7 +41,13 @@ class PhoneNumberTypeTest extends TestCase
 
     public static function setUpBeforeClass()
     {
-        Type::addType('phone_number', 'Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType');
+        if (method_exists('Symfony\\Component\\Form\\FormTypeInterface', 'getName')) {
+            $type = new PhoneNumberType();
+        } else {
+            $type = 'Misd\\PhoneNumberBundle\\Form\\Type\\PhoneNumberType';
+        }
```

Comment:
Not sure what you are trying to do here, but this is a test for a Doctrine type and not for a form type.